bundles/org.dataflowanalysis.analysis.dfd/src/org/dataflowanalysis/analysis/dfd/dsl/DFDVertexType.java

```diff
@@ -32,4 +34,20 @@ public boolean matches(AbstractVertex<?> vertex) {
             }
         }
     }
+
+    public static ParseResult<VertexType> fromString(StringView string) {
```

Comment:
In general: At least some Javadoc for such high level methods?

Reply:
Fixed in 745e3a1b5bfb